### ISSUE TITLE
Share Innovus map file

### ISSUE THREAD

**Issue by egiacomin** [google/skywater-pdk#313]
I just finished a design and created the Innovus map file by hand to be able to export my .gds and import it sucessfully on Virtuoso. I've checked the _skywater-pdk/vendor/synopsys/PlaceRoute/common_ but none of the provided file were compatible with Innovus.
What would be the best way to share this file? Should I do a pull request and add it to the _skywater-pdk/vendor/synopsys/PlaceRoute/common_ folder?

**Comment by nguyendao-uom** [comment]
Hi @egiacomin, I'm also looking for the Innovus map file. Can you share it here? Thanks

**Comment by egiacomin** [comment]
Here is the one I uses. Let me know if you run into any issues when using it!

```
# CADENCE Stream layer mapping table for Skywater130nm
# <t_ layerObjectName> <t_ objectType> <n_ layerNum> <n_ datatype>

poly		NET, SPNET, VIA 	66 	20
poly		PIN 							66 	16
licon1	VIA 							66 	44
licon1	PIN 							66 	41
licon1	 NET 							66 	58
li1			NET, SPNET, VIA 	67 	20
li1			PIN 							67 	16
mcon		VIA 							67 	44
mcon		PIN 							67 	41
mcon		NET 							67 	58
met1		NET, SPNET, VIA 	68 	20
met1		PIN 							68 	16
via			VIA 							68 	44
via			PIN 							68 	41
via			NET 							68 	58
met2		NET, SPNET, VIA 	69 	20
met2		PIN 							69 	16
via2		VIA 							69 	44
via2		PIN 							69 	41
via2		NET 							69 	58
met3		NET, SPNET, VIA 	70 	20
met3		PIN 							70 	16
via3		VIA 							70 	44
via3		PIN 							70 	41
via3		NET 							70 	58
met4		NET, SPNET, VIA 	71 	20
met4		PIN 							71 	16
via4		VIA 							71 	44
via4		PIN 							71 	41
via4		NET 							71 	58
met5		NET, SPNET, VIA 	72 	20
met5		PIN 							72 	16
```

**Comment by mithro** [comment]
We should totally figure out a process for getting this into the SkyWater PDK....

**Comment by nguyendao-uom** [comment]
Thanks @egiacomin, It seems OK for exporting gds file from Innovus, but doesn't show up all drawing layers (only Pin and via layers) when stream in Virtuoso. Do you have a tech file for importing the gds file?

**Comment by mithro** [comment]
BTW There is a `#cadence-innovus` channel on the [SkyWater Slack channel](https://join.skywater.tools/) that might be able to help.

**Comment by egiacomin** [comment]
@nguyendao-uom I have access to the original PDK with the technology library, so I simply created a new library, attached it to the skywater technology_library and imported my GDS without any layer mapping file. What procedure did you follow?

@mithro should I do a pull request to add the file under the _skywater-pdk/vendor/synopsys/PlaceRoute/common_ directory? Or do you have something else in mind?

**Comment by nguyendao-uom** [comment]
@egiacomin : you mean the s8 pdk from skywater? I don't have the access to it.

**Comment by egiacomin** [comment]
@nguyendao-uom yes. Have you been able to import any standard cell GDS successfully by any chance? If yes, what mapfile/techfile did you use?

**Comment by nguyendao-uom** [comment]
We have TSMC, UMC technologies only. And, of course, we have the NDA for the access.

**Comment by mattvlsi** [comment]
Hi @egiacomin. I'm trying to import SOFA FPGA's gds files into Virtuoso or Custom Compiler. By using your map file it successfully imports the gds. And there seems correct layer names and colours. However in layout there are only some vias and small rectangulars. Unable to see standart cells of sky130 and top view of SOFA FPGA's. Can you describe more detailed way how to attach tecnology file and where can I find this technology library? Thanks.

**Comment by m-usama-z** [comment]
Hi. I've a question about the layer map file used for exporting GDS from Innovus:
There are some purpose numbers mentioned in the skywater layer info excel sheet, which are not mentioned in the above layer map file, e.g. 13, 10, 60, 0, 14, 25 and 15. Would the absence of these purpose numbers in the map file cause issues in fabrication?

**Comment by m-usama-z** [comment]
Kindly add the following line to the map file, whenever it may be incorporated into the PDK:

DIEAREA         ALL       235    4

This is required for density check using Klayout.

**Comment by FelipeWcosta** [comment]
Hi @egiacomin!

I am having issues with the layer map file. I would like to ask if is it possible run Rail Analysis using your layer map file using Innovus and Voltus?

Kind regards.
